Adyen/Core/ToolBar/CancellableToolBar.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+@_spi(AdyenInternal)
+public class CancellableToolBar: ModalToolbar {
+
+    private let style: NavigationStyle
+    private let title: String?
+
+    public override init(title: String?, style: NavigationStyle) {
+        self.style = style
+        self.title = title
+        super.init(title: title, style: style)
+    }
+
+    private func customizeCancelButton() -> UIButton {
+        let button = UIButton(type: UIButton.ButtonType.system)
+        let cancelText = Bundle(for: UIApplication.self).localizedString(forKey: "Cancel", value: nil, table: nil)
+        button.setTitle(cancelText, for: .normal)
+        button.titleLabel?.font = .systemFont(ofSize: 17.0)
```

Comment:
Is this necessary? What font is it by default?

Reply:
Yes this is necessary.